rasgotransforms/rasgotransforms/macros/filter.sql

```diff
@@ -19,6 +19,12 @@
     {% if filter is not string and filter is not mapping %}
     {% set filter = dict(filter) %}
     {% endif %}
+    {% if 'column_name' in filter %}
```

Comment:
I would invert this logic and replace all instance of camelCase vars with snake_case vars in this package. This is an open-source python package, and as such we should be unapologetic about following python standards. We should use snake in every macro and template and require consumers (our UI) to make case translations when passing in values